docs/content/modules/player-lifecycle-feature-module.md

```diff
@@ -14,32 +14,40 @@ Before reading this document, make sure you are familiar with:
 
 The Player lifecycle module is a feature module providing you with a simple player spawning and player lifecycle management implementation.
 
-### How to enable this module
+## How to enable this module
```

Comment:
intentional?

Reply:
I think so - it looks like it was off by one originally.